daemon/daemon.go

```diff
@@ -782,9 +782,11 @@ func NewDaemon(config *Config, registryService *registry.Service) (daemon *Daemo
 		return nil, fmt.Errorf("Couldn't restore custom images: %s", err)
 	}
 
+	migrationStart := time.Now()
 	if err := v1.Migrate(config.Root, d.driver.String(), d.layerStore, d.imageStore, tagStore, distributionMetadataStore); err != nil {
 		return nil, err
 	}
+	logrus.Debugf("migration took %.2f seconds", time.Since(migrationStart).Seconds())
```

Comment:
Timing stats could be useful to have at the info level if we're trying to track down performance issues with migration. The log line should only ever be displayed once, so I don't think it would be overly verbose to have it at the info level.